131 finish to setup phpstan

ctresvaux opened the pull request:
## Related issue(s)
<!-- Copy-paste the URL to the related issue(s) if any ("N/A" if not applicable).
  -->

https://github.com/Probesys/agentj/issues/131

## How to test manually
<!-- List actions (step by step) of what have to be done in order to test your
  -- changes manually ("N/A" if not applicable).
  -->

N/A

## Reviewer checklist
<!-- Please don't change or remove this checklist. It will be used by the
  -- reviewer to make sure to not forget important things.
  -- Reviewer: if you think one of the item isn’t applicable to this PR,
  -- please check it anyway.
  -->

- [x] Code is manually tested
- [x] Interface works on both mobiles and big screens
- [x] Interface works on both Firefox and Chrome
- [x] Tests are up to date
- [x] Documentation is up to date
- [x] Pull request has been reviewed and approved

ctresvauxp6 committed:
chore(app): Finish to configure PHPStan to level 6

marien-probesys commented on app/src/Command/UpdateGroupsWblistCommand.php:
```diff
@@ -36,20 +36,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $rootMailaddr = $this->em->getRepository(Mailaddr::class)->findOneBy((['email' => '@.']));
         foreach ($activeGroups as $group){
             $groupsWblist = $this->em->getRepository(GroupsWblist::class)->findOneBy((['mailaddr' => $rootMailaddr, 'groups' => $group]));
-            if (!$groupsWblist){
```
Je ne pense pas qu’il faille supprimer cette condition : `findOneBy` peut théoriquement retourner `null`

marien-probesys committed:
chore(ci): Fix the CI for PHPStan

PHPStan needs the var/cache/dev/App_KernelDevDebugContainer.xml file to
exists, which is built through compose install when it is executed with
scripts. However, the .env file needs to be valid for that, so this is
why we create it.
